back/config/routes.rb

```diff
@@ -87,6 +87,7 @@
         get 'initiatives_count', on: :member
         get 'comments_count', on: :member
         get 'blocked_count', on: :collection
+        get 'check/:email', on: :collection, to: 'users#check', constraints: { email: /.*/ }
```

Comment:
Is the constraint necessary? I guess it is, but I'm a bit confused bc it matches with everthing.

Reply:
Odd one this - the constraint allows us to put a full email address in the url - without it, rails chops off anything past the dot.